cmd/juju/payload/list_test.go

```diff
@@ -14,8 +14,8 @@ import (
 	jc "github.com/juju/testing/checkers"
 	gc "gopkg.in/check.v1"
 
-	"github.com/juju/juju/payload"
-	"github.com/juju/juju/payload/status"
+	"github.com/juju/juju/cmd/juju/payload"
+	corepayload "github.com/juju/juju/payload"
```

Comment:
if we are going to call it core payload and it fits what is in core/ then maybe we move it out of root into juju/juju/core/

Reply:
That's scheduled for the last PR after everything else; trying to manage the size of the changes each time.